layers/transportation_name/transportation_name.sql

```diff
@@ -28,9 +28,9 @@ CREATE OR REPLACE FUNCTION layer_transportation_name(bbox geometry, zoom_level i
 AS
 $$
 SELECT geometry,
-       name,
-       COALESCE(name_en, name) AS name_en,
-       COALESCE(name_de, name, name_en) AS name_de,
+       tags->'name' AS name,
+       COALESCE(tags->'name_en', tags->'name') AS name_en,
+       COALESCE(tags->'name_de', tags->'name', tags->'name_en') AS name_de,
```

Comment:
Shouldn't these be `name:en` and `name:de`?
```suggestion
       COALESCE(tags->'name:en', tags->'name') AS name_en,
       COALESCE(tags->'name:de', tags->'name', tags->'name:en') AS name_de,
```

When as it is now `tags->'name_en` and `tags->'name_de'` are taken from `osm_transportation_name_linestring_genX` and `osm_transportation_name_linestring`. 

Where `osm_transportation_name_linestring` is created from `osm_transportation_name_network` and `osm_shipway_linestring` where is using new function `transportation_name_tags` which creating tags `name`, `name:en` and `name:de`?